Solutions/Synack/Integrations/AzureFunctionSynack/sync-service.js

```diff
@@ -0,0 +1,186 @@
+                                                for (let incidentComment of incidentCommentsJsonArray) {
+                                                    incidentCommentsIds.push(incidentComment['name'])
+                                                }
+                                                for (let synackComment of synackCommentsJsonArray) {
+                                                    let sentinelCommentId = `${incidentId}_${synackComment.id}`
+                                                    if (!incidentCommentsIds.includes(sentinelCommentId)) {
+                                                        let synackAuthorName = synackComment['User']['name']
+                                                        let sentinelCommentBody = `**${synackAuthorName}** **(Synack):** ${synackComment.body}`
+                                                        azureService.createComment(incidentId, sentinelCommentId, sentinelCommentBody, accessToken)
+                                                            .then((commentOperationResult) => {
+                                                                    console.log(`${commentOperationResult.status} comment ${commentOperationResult.name}`)
+                                                                }
+                                                            )
+                                                            .catch((error) => {
+                                                                console.error(`Failed to create comment for incident ${incidentId}`)
+                                                                console.error(error)
+                                                            })
+                                                    } else {
+                                                        console.log(`comment ${sentinelCommentId} already exists`)
+                                                    }
+                                                }
+                                            })
+
+                                    })
+                            })
+                            .catch((error) => {
+                                console.error('error occurred while trying to create/update a Sentinel incident')
+                                console.error(error)
+                            })
+                    }
+                })
+                .catch((error) => {
+                    console.error(`error occurred while trying to get Azure authentication token \n ${error}`)
+                })
+        })
+        .catch((error) => {
+            console.error(`The synchronization failed. Error occurred while trying to fetch vulnerabilities from Synack: ${error}`)
+        })
+}
+
+function getIncidentStatus(synackStatusJson) {
+    // https://docs.microsoft.com/en-us/rest/api/securityinsights/incidents/create-or-update#incidentclassification
```

Comment:
@andriykozynets :  PR Validation checks are failing because the link mentioned here contains locale (en-us), please update the link to pass the validations.... Thanks!!!

Reply:
Done! @v-rucdu